docs/static/releasenotes.asciidoc

```diff
@@ -36,6 +37,39 @@ This section summarizes the changes in the following releases:
 * <<logstash-7-0-0-alpha2,Logstash 7.0.0-alpha2>>
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
+[[logstash-7-10-1]]
+=== Logstash 7.10.1 Release Notes
+
+==== Notable issues fixed
+
+===== Support recreation of same pipeline through centralized pipeline management
+When users attempted to delete and recreate a pipeline with the same identifier and configuration, Logstash was unable
+to pick up the new pipeline. https://github.com/elastic/logstash/issues/12387[#12387]
+
+==== Plugins
+
+*Azure_event_hubs Input - 1.2.3*
+
+* Fixed missing configuration of `prefetch_count` and `receive_timeout` https://github.com/logstash-plugins/logstash-input-azure_event_hubs/pull/61[#61]
+
+*Http Input - 3.3.6*
+
+* Fixes a regression introduced in **3.1.0**'s migration to the Netty back-end that broke some users'
+   browser-based workflows. When a plugin configured to require Basic authentication receives a request that does not
```

Comment:
```suggestion
   browser-based workflows for some users. When a plugin that is configured to require Basic authentication receives a request that does not
```

Reply:
The use of `'` in `**3.1.0**'s` and `users'` is causing the text between the characters to render in italics.  The inline suggestions rework the text to avoid the formatting problem. 
Note: Sometimes suggestions across multiple line need spacing tweaks after they are accepted.

Reply:
Thanks @karenzone, suggestions included